views/drones/list.hbs

```diff
@@ -1 +1,32 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<div>
+
+  <h2 class="title">List of available drones</h2>
+
+  <hr>
+
+  <div class="drones">
+    {{#each DronesFromDB}}
+
+    <div class="dronebox">
+      
+      <h3>{{name}}</h3>
+      <div class="middledronebox">Number of propellers: {{propellers}}. </div>
+      <div class="speed-drone" >Max. speed: {{maxSpeed}} m per second.</div>
+      <form action="drones/{{_id}}/edit" method="GET">
```

Comment:
You could also use an anchor element `<a href ="/drones/{{_id}}/edit"> Edit drone</a>`, you should only use a form if you're looking to pass data along your GET request